common/travis/license.lst

```diff
@@ -58,12 +59,14 @@ BSD-Protection
 BSD-Source-Code
 BSL-1.0
 BUSL-1.1
+Baekmuk
 Bahyph
 Barr
 Beerware
 Bison-exception-2.2
 BitTorrent-1.0
 BitTorrent-1.1
+Bitstream-Vera
```

Comment:
try to find packages that use this license and remove custom: from their $license field

Reply:
Done, I hope I committed the changes correctly?

Reply:
lgtm